clicache/src/impl/PdxFieldType.cpp

```diff
@@ -166,23 +166,23 @@ namespace Apache
           switch (m_typeId)
           {
           case PdxFieldTypes::BYTE:
-            return PdxTypes::BYTE_SIZE;
+            return PdxTypes::PDX_BYTE_SIZE;
```

Comment:
If you are changing these then we should be changing them to match the style guide we currently have approved. In our last discussion about the style guide we did not agree on a deviation for enum names. By using prescribed naming convention you will avoid the macro name collision.

https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
Done. Nice catch!